docs/commerce/2.x/en/managing-a-catalog/creating-and-managing-products/channels/introduction-to-channels.md

```diff
@@ -1,22 +1,12 @@
 # Introduction to Channels
 
-Liferay Commerce provides tools designed to meet the needs of complex B2B, multi-region, and multi-market stores and simpler B2C use cases. One way that Commerce enables a wide variety of use cases is through the _Channels_ feature.
+Channels represent a place where products are sold, whether that's a Liferay Commerce based customer portal, a B2B marketplace, or back office phone sales. By connecting sales channels to Liferay Commerce, sellers can manage product content, inventory, orders, and fulfillment across multiple channels in a single place.
 
-In Liferay Commerce, Channels connect Catalogs and Products to any place products are sold: a Liferay Commerce based storefront, third party marketplaces, or back office phone sales. Using Channels, users manage product availability, sales, and fulfillment by Site and eventually by Country, Region, or Account Group. Channels must be associated to warehouses in order to track product availability and and sourcing for product shipping.
+To display products and receive orders, a virtual instance must have at least one channel.
 
-In a simple store example, all of a company's products are managed in one catalog and those products are distributed through a single channel: the company store site.
+With multiple channels, you can create more than one Liferay site each with a unique customer experience, connect to third party software or systems, or a combination of both.
 
-![Simple store structure](./introduction-to-channels/images/01.png)
```

Comment:
Needs to be updated. Like I've mentioned before, my diagram is not what was actually implemented. There is no direct relationship between a catalog and a channel. The relationship is between products and channels.